src/Root.tsx

```diff
@@ -29,7 +32,6 @@ export const wrapPageElement: GatsbyBrowser['wrapPageElement'] = ({
 }) => {
   return <Layout>{element}</Layout>;
 };
-
```

Comment:
Why remove this? It's good to have some empty lines between different functions for readability.

Reply:
indeed, it should keep 1 line space, let me double check the whole PR if contains such behaviour or not

Reply:
resolved